static/app/components/replays/breadcrumbs/timelineGaps.tsx

```diff
@@ -13,76 +14,71 @@ import {
 interface Props {
   durationMs: number;
   frames: ReplayFrame[];
-  totalFrames: number;
-  width: number;
+  startTimestampMs: number;
 }
 
 // create gaps in the timeline by finding all columns between a background frame and foreground frame
 // or background frame to end of replay
-export default function TimelineGaps({durationMs, frames, totalFrames, width}: Props) {
-  const markerWidth = totalFrames < 200 ? 4 : totalFrames < 500 ? 6 : 10;
-  const totalColumns = Math.floor(width / markerWidth);
-  const framesByCol = getFramesByColumn(durationMs, frames, totalColumns);
-
-  // returns all numbers in the range, exclusive of start and inclusive of stop
-  const range = (start, stop) =>
-    Array.from({length: stop - start}, (_, i) => start + i + 1);
-
-  const gapCol: number[] = [];
-
-  const gapFrames = framesByCol.entries();
+export default function TimelineGaps({durationMs, startTimestampMs, frames}: Props) {
+  const ranges: Array<{left: string; width: string}> = [];
+  const gapFrames = frames.entries();
   let currFrame = gapFrames.next();
 
   while (!currFrame.done) {
```

Comment:
This could probably become more like `for (const currFrame of gapFrames) {` and we can avoid doing the `.next()` and `.done` calls manually.